suite-common/validators/src/schemas/sendFormSchema.ts

```diff
@@ -0,0 +1,121 @@
+import { G } from '@mobily/ts-belt';
+import BigNumber from 'bignumber.js';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { formatNetworkAmount, isAddressValid } from '@suite-common/wallet-utils';
+import { FeeInfo } from '@suite-common/wallet-types';
+
+import { yup } from '../config';
+
+export type SendFormFormContext = {
+    networkSymbol?: NetworkSymbol;
+    availableAccountBalance?: string;
+    networkFeeInfo?: FeeInfo;
+};
+
+const isAmountDust = ({
+    value,
+    networkSymbol,
+    isValueInSats,
+    networkFeeInfo,
+}: {
+    value: string;
+    networkSymbol: NetworkSymbol;
+    isValueInSats: boolean;
+    networkFeeInfo: FeeInfo;
+}) => {
+    const valueBigNumber = new BigNumber(value);
+    const rawDust = networkFeeInfo.dustLimit?.toString();
+
+    const dustThreshold =
+        rawDust && (isValueInSats ? rawDust : formatNetworkAmount(rawDust, networkSymbol));
+
+    if (!dustThreshold) {
+        return false;
+    }
+
+    return valueBigNumber.lt(dustThreshold);
+};
+
+const isAmountHigherThanBalance = ({
```

Comment:
There are no functions for this that could be reused in utxo lib or desktop implementation?

Reply:
This is inspired by `packages/suite` implementation of validation. My original idea was to extract it to these new functions and the import it to desktop. But the desktop implementation is so messed up that the inputs and outputs of the functions would have to be way more complicated than it is now. For this reason I decided to not refactor it.

On the other hand if the desktop will refactor the send form one day, they can easily use this scheme from since it is defined in `suite-common.`